packages/jsxstyle-utils/src/getStyleCache.ts

```diff
@@ -16,7 +17,8 @@ function alreadyInjected() {
   );
 }
 
-function getStringHash(key: string, props: {}) {
+function getStringHash(key: string, props: {}): string;
+function getStringHash(key: string) {
   return '_' + stringHash(key).toString(36);
 }
 
```

Comment:
`getStyleCache` is missing a return type annotation

Reply:
Yeah, it’s inferred in the second case.

Reply:
Thought about this a bit more. It’s probably best to be explicit.